Dockerfile

```diff
@@ -1,18 +1,35 @@
-FROM node:15 as builder
-LABEL author="chevdor@gmail.com"
```

Comment:
We should only add authors/maintainers, not remove them. ref:

https://github.com/paritytech/substrate-telemetry/pull/269#discussion_r455126513

Reply:
:+1: